### ISSUE TITLE
`buildifier --format_bzl` fails if a loop's first line is a comment

### ISSUE THREAD

**Issue by jmillikin-stripe** [bazelbuild/buildtools#206]
Using current head (b02a20ba60456ba571a72bf66f45b3955f757965):
```
$ cat foo.bzl 
def foo():
  for src in []:
    # x
    pass
$ buildifier --format_bzl foo.bzl 
foo.bzl:3:8: syntax error near comment
$
```

**Comment by laurentlb** [comment]
To set expectations: the option `--format_bzl` is still under development, and it cannot parse every bzl file yet. So expect more bugs like this.

But thanks for the report, it's useful to get early feedback!

**Comment by jmillikin-stripe** [comment]
Yep, saw the comments about it being experimental. I'm excited to have this feature but I don't know enough about parsing Skylark to help implement, so I figured I could at least add some test cases.

**Comment by vladmos** [comment]
FYI: the `--format_bzl` has been replaced with `--type=bzl`.
